aspnetcore/performance/caching/middleware.md

```diff
@@ -1,15 +1,118 @@
 ---
-title: 🔧 Response caching middleware
+title: Response Caching Middleware | Microsoft Docs
 author: rick-anderson
+description: Configuration and use of Response Caching Middleware in ASP.NET Core applications.
+keywords: ASP.NET Core, response caching, caching, ResponseCache, ResponseCaching, Cache-Control, VaryByQueryKeys, middleware
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 12/15/2016
 ms.topic: article
 ms.assetid: f9267eab-2762-42ac-1638-4a25d2c9d67c
 ms.prod: aspnet-core
 uid: performance/caching/middleware
 ---
-# 🔧 Response caching middleware
+# Response Caching Middleware
 
-> [!NOTE]
-> We are currently working on this topic.We welcome your input to help shape the scope and approach. You can track the status and provide input on this [issue](https://github.com/aspnet/Docs/issues/2198) at GitHub.If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the [issue](https://github.com/aspnet/Docs/issues/2198). Learn more about how you can [contribute](https://github.com/aspnet/Docs/blob/master/CONTRIBUTING.md) on GitHub.
+By [Luke Latham](https://github.com/GuardRex) and [John Luo](https://github.com/JunTaoLuo)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/middleware/sample)
+
+This document provides details on how to configure the Response Caching Middleware in ASP.NET Core applications. The middleware determines when responses are cacheable (primarily based on response headers you set), caches responses, and serves responses from cache when appropriate. For an introduction to HTTP caching and the `ResponseCache` attribute, see [Response Caching](response.md).
```

Comment:
I'll let @Rick-Anderson work on the language for the doc but here's my take. "When appropriate" sounds ambiguous, let's leave it out. "caches responses" => "stores responses", "(primarily based on response headers you set)" => "using headers, options and features configured by the application", though I suppose you can keep the "you" part if you really want it.